flight/apps/comms/rf_mcu.py

```diff
@@ -0,0 +1,39 @@
+"""
+
+High-level interface to communicate with the RF MCU managing the SEMTECH SX1262 RF LoRa module (E22-900M30S).
+It defines the SPI communication protocol between the 2 MCU's.
+
+TODO: Until porting to new board configuration is complete, this will send directly to the radio module.
+
+"""
+
+from hal.configuration import SATELLITE
+
+
+class RF_MCU:
+    """
+    High-level interface to communicate with the RF MCU managing the SEMTECH SX1262 RF LoRa module (E22-900M30S).
+    """
+
+    SPI = None
+
+    @classmethod
```

Comment:
Why are these `@classmethods`? Shouldn't the satellite initialize one instance of RF_MCU and that instance will use these methods?

Reply:
Same comment as above (class variable removed, not needed) + no need for instance behavior (not even shared state this time) and methods will be accessed in other places. The SPI interface will be defined in SATELLITE anyway.